bertytech/content/docs/protocol/index.md

```diff
@@ -146,12 +146,12 @@ func rendezvousPoint(id, seed []byte, date time.Time) []byte {
 
 There are two types of rendezvous points in the Wesh protocol:
 
-* [**Public rendezvous point:**](#contact-request) This rendezvous point is
+* [**Public rendezvous point:**](docs/protocol/#contact-request) This rendezvous point is
```

Comment:
Yes, this is the brute-force way to fix it. Of course, now the inner page references don't work in the original Markdown file:
https://github.com/berty/www.berty.tech/blob/9b4b53e084578b7f5ee48d2115613aaca315c3b1/bertytech/content/docs/protocol/index.md

The long-term solution is to understand why all page references move to the root when compiling as HTML. But for now, it is more important that the document on the web page works properly, so I approve merging this change.